src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/steps/VotingStep/partials/TextWithValue/TextWithValue.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+
+import { TextWithValueProps } from './types';
+
+const TextWithValue: FC<TextWithValueProps> = ({ items }) => (
+  <dl className="mb-6">
```

Comment:
This margin shouldn't be added here. Please add `className` prop and use it to add the margin later